client/src/clients/api/vacations.ts

```diff
@@ -39,6 +39,15 @@ export class VacationsApi extends ApiClientBase {
     return requests
   }
 
+  async approveOrRejectAllTeamPendingRequets(data: Api.ApproveOrRejectAllTeamPendingRequets) {
+    ApiClientBase.assertUser()
+    const requests = await this.unwrap(
+      () => this.$http.put<Api.Returns.List<Api.Vacation>>(this.getUrl('/action-team-peneding-requests'), data),
```

Comment:
```suggestion
      () => this.$http.put<Api.Returns.List<Api.Vacation>>(this.getUrl('/action-team-pending-requests'), data),
```

Also, should be fixed in 'my-peneding-requests' & 'action-team-peneding request' too.